solidity/contracts/bridge/Wallets.sol

```diff
@@ -569,4 +569,33 @@ library Wallets {
 
         beginWalletClosing(self, walletPubKeyHash);
     }
+
+    /// @notice Called when the timeout for MovingFunds wallet elapsed.
+    ///         Slashes wallet members and terminates the wallet.
+    /// @dev Requirements:
+    ///      - The wallet must be in the MovingFunds state.
```

Comment:
Let's document the parameters as well.

Reply:
I added those comments from the commit view so this probably caused confusion. You are right, in the final version, this is the only occurrence.